data/strings/strings.txt

```diff
@@ -2584,8 +2584,7 @@
   [fuel]
     comment = Search category for fuel stations; any changes should be duplicated in categories.txt @fuel!
     tags = android,ios
-    en = Gas
-    en-GB = Petrol
+    en = Fuel
```

Comment:
Nobody says or types Fuel neither in the US nor in the UK.

Reply:
my concern was people in en-GB getting confused that there wasn't a 'Diesel' category, but after considering it, i think it's best to leave as it was 👍

Reply:
yeah, i think what it was before will do, or at least it'll require more consideration :)